1_2.js

```diff
@@ -0,0 +1,10 @@
+function solution(num) {
+    if (num%2){
```

Comment:
```suggestion
    if (num % 2) {
```
보기쉽게 컨벤션 지켜서 작성 해 주세요